core/comment.ts

```diff
@@ -277,21 +272,8 @@ export class Comment extends Icon {
    * Dispose of the bubble.
    */
   private disposeBubble() {
-    if (this.onMouseUpWrapper) {
-      browserEvents.unbind(this.onMouseUpWrapper);
-      this.onMouseUpWrapper = null;
-    }
-    if (this.onWheelWrapper) {
-      browserEvents.unbind(this.onWheelWrapper);
-      this.onWheelWrapper = null;
-    }
-    if (this.onChangeWrapper) {
-      browserEvents.unbind(this.onChangeWrapper);
-      this.onChangeWrapper = null;
-    }
-    if (this.onInputWrapper) {
-      browserEvents.unbind(this.onInputWrapper);
-      this.onInputWrapper = null;
+    for (const event of this.boundEvents) {
+      browserEvents.unbind(event);
```

Comment:
I'm not sure we actually need to unbind these. I learned recently that listeners should be garbage collected when the element the listener is associated with is.

Similarly for the rest of commit e965089c28da5caf850b5f17cac89c2ca6f8b13b

Reply:
Historically the HTML world and the JavaScript world had separate garbage collectors which didn't talk to each other.  This meant if an element on one side linked to an element on the other side, the garbage collectors were powerless to clean up.  Thus one had to sever the links manually upon disposal.

Whether this is still true today is unknown to me.  Last month I ran a series of tests on this, but Chrome leaks memory so badly that it was impossible to determine how its garbage collector works.  And of course, we'd need to verify that _all_ supported browsers have upgraded garbage collectors before we could strip down our dispose functions.

Reply:
I tested this locally when I was working https://github.com/google/blockly/issues/6890. I ran the following steps, both with and without unbinding events:

1) Start a performance recording (with memory recording enabled).
2) Trigger the spaghetti test on the playground.
3) Force a GC.
4) Delete the top block in the spaghetti stack.
5) Force a GC.
6) Stop the performance recording.

This resulted in the same number of listeners after the final GC (both with and without unbinding).

All browsers ship with mark-and-sweep, so I can't imagine this is different cross browser. All of my searching also states that this is generally applicable (excepting IE ofc).

However, since this PR is so large and we want to get this merged before Friday. I think we can just leave as is and I can look into it as part of #6890